### PR TITLE
Update supporter info for README.md.

### PR DESCRIPTION
**Pull request by digisomni**

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-PR1293-dc84886-.tar_11e1e0c2.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-PR1293-dc84886-_11e1e0c2.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-PR1293-dc84886-.tar_11e1e0c2.Z

**build (macOS-latest, client)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-Interface-PR1293-dc84886-_11e1e0c2.dmg

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-PR1293-dc84886-_11e1e0c2.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1293/Vircadia-PR1293-dc84886-_11e1e0c2.exe

File: README.md [hunk 1]
```diff
@@ -73,8 +73,16 @@ Vircadia consists of many projects and codebases with its unifying structure's g
 
 There are many contributors to Vircadia. Code writers, reviewers, testers, documentation writers, modelers, and general supporters of the project are all integral to its development and success towards its goals. Find out how you can [contribute](CONTRIBUTING.md)!
 
-### [Supporters](https://github.com/sponsors/digisomni/)
+### Support
 
-| [Karol Suprynowicz - 74hc595](https://github.com/ksuprynowicz) | 
+You can support the Vircadia project financially through [GitHub Sponsors](https://github.com/sponsors/digisomni/).
```
Comment (JulianGro) [inline README.md:78]:
```suggestion

```
I don't see how sponsoring you is supporting the Vircadia project, but sponsoring David or me isn't.

File: README.md [hunk 2]
```diff
@@ -73,8 +73,16 @@ Vircadia consists of many projects and codebases with its unifying structure's g
...
+
+You can also support individual active contributors by visiting each repository, this is the main Vircadia repository's [list of active contributors](https://vircadia.com/contributors-redirector/?redirect=vircadia/vircadia). Click on a contributor's profile to see if they accept donations!
```
Comment (JulianGro) [inline README.md:80]:
```suggestion
You can support individual active contributors by visiting each repository, this is the main Vircadia repository's [list of active contributors](https://vircadia.com/contributors-redirector/?redirect=vircadia/vircadia). Click on a contributor's profile to see if they accept donations!
```

**Review by JulianGro — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (digisomni) [reply README.md:80]: I'm the project's founder so I am fully invested.

I reimburse for hosting fees where possible. I also pay a lot of money for licensing, hosting, domains, and other stuff from day to day, month to month, year to year.

Also I pay people (or are getting into it) to try and get things done in and around the project.

I have and always do reinvest money.

**Pull request merged**